worlds/terraria/Options.py

```diff
@@ -1,40 +1,76 @@
 from dataclasses import dataclass
-from Options import Choice, DeathLink, PerGameCommonOptions
+from Options import Choice, DeathLink, PerGameCommonOptions, Toggle
+
+
+class Calamity(Toggle):
+    """Calamity mod bosses and events are shuffled"""
+
+    display_name = "Calamity Mod Integration"
+    default = False
+
+
+class Getfixedboi(Toggle):
+    """Generation accomodates the secret, very difficult "getfixedboi" seed"""
+
+    display_name = """"getfixedboi" seed"""
+    default = False
```

Comment:
```suggestion
class Calamity(Toggle):
    """Calamity mod bosses and events are shuffled"""

    display_name = "Calamity Mod Integration"


class Getfixedboi(Toggle):
    """Generation accomodates the secret, very difficult "getfixedboi" seed"""

    display_name = """"getfixedboi" seed"""
```
Don't need to put a default for Toggles -- they default to False